code-studio/src/js/components/AdvancedShareOptions.jsx

```diff
@@ -94,15 +120,31 @@ var AdvancedShareOptions = React.createClass({
     var optionsNav;
     var selectedOption;
     if (this.state.expanded) {
+      var selectedLiStyle = Object.assign({}, style.nav.li, {color:color.purple});
```

Comment:
Does `Object.assign` get polyfilled or transpiled?  It [doesn't seem to be supported](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Browser_compatibility) across our whole browser matrix.